docs/pages/contractsAndAudits.md

```diff
@@ -2,10 +2,11 @@
 
 | Name    | Address    | 
 |-------------|-------------|
-| Nexus implementation  | 0x000000039dfcAd030719B07296710F045F0558f7  |
-| K1 Validator  | 0x00000004171351c442B202678c48D8AB5B321E8f  | 
-| Bootstrap  | 0x00000008c901d8871b6F6942De0B5D9cCf3873d3  |
-| K1 Validator Factory  | 0x00000bb19a3579F4D779215dEf97AFbd0e30DB55  | 
+| Nexus implementation  | 0x000000008761E87F023f65c49DC9cb1C7EdFEaaf  |
```

Comment:
@filmakarov is this definitely correct? I have a different address in the SDK

Reply:
It is correct.
This is the new one, I deployed it today
So you need to add this to your SDK

I re-deployed all the contracts today, so the new addresses need to be used on SDK right"?